packages/jellyfish-api-core/__tests__/category/wallet/getUnconfirmedBalance.test.ts

```diff
@@ -0,0 +1,24 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BigNumber } from '../../../src'
```

Comment:
Use this `import BigNumber from 'bignumber.js'`
If you see `import { BigNumber } from '../../../src'` somewhere, they need to be refactored one day